webUI/react/src/components/navbar/ReaderNavBar.tsx

```diff
@@ -141,6 +141,7 @@ export const defaultReaderSettings = () => ({
     continuesPageGap: false,
     loadNextonEnding: false,
     readerType: 'ContinuesVertical',
+    readerDirection: 'LeftToRight',
```

Comment:
`Continues Vertical` + `LeftToRight`? this doesn't make sense

Maybe make it `Continues` + `Vertical`?

Reply:
What `Continues Vertical` + `LeftToRight` does is it moves to `next/prev` page when `right/left` arrow keys are pressed.

And `Continues Vertical` + `RightToLeft` does the opposite when the arrow keys are pressed.